doc/nrf/libraries/bin/lwm2m_carrier/CHANGELOG.rst

```diff
@@ -18,6 +18,49 @@ For certification status of the released versions, see the following documents,
 * `nRF9161 Mobile network operator certifications`_
 * `nRF9160 Mobile network operator certifications`_
 
+liblwm2m_carrier 3.6.0
+**********************
+
+Release for modem firmware version 1.3.7 and 2.0.2.
+
+Size
+====
+
+See :ref:`lwm2m_lib_size` for an explanation of the library size in different scenarios.
+
++-------------------------+---------------+------------+
+|                         | Flash (Bytes) | RAM (Bytes)|
++-------------------------+---------------+------------+
+| Library size            |               |            |
+| (binary)                |               |            |
++-------------------------+---------------+------------+
+| Library size            |               |            |
+| (reference application) |               |            |
++-------------------------+---------------+------------+
+
+Changes
+=======
+
+* Removed the configuration ``disable_bootstrap_from_smartcard`` (and corresponding Kconfig ``LWM2M_CARRIER_BOOTSTRAP_SMARTCARD``).
+
+  * Instead, the LwM2M carrier library will bootstrap from smartcard if the external :ref:`lib_uicc_lwm2m` library is included in the project. (:kconfig:option:`UICC_LWM2M`)
+
+* Changed the CoAP response if an unsupported image is pushed during in-band FOTA.
+
+  * Previous CoAP return value was ``4.00 Bad Request``, and is now ``5.00 Internal Server Error``.
+
+* Changed the timing of the lifetime update in order to better allow CoAP retransmissions. (Effectively, the updates will now trigger earlier.)
+
+  * This can reduce the amount of overall signalling needed.
+    For example: in poor network conditions, the Server could have de-registered the client prematurely, when in fact the client was still retransmitting and attempting to update with the server.
```

Comment:
```suggestion
  * This can reduce the amount of overall signaling needed.
    For example, in poor network conditions, the server could have de-registered the client prematurely, when the client was still retransmitting and attempting to update with the server.
```